google/cloud/storage/__init__.py

```diff
@@ -31,10 +31,12 @@
   machine).
 """
 
+import pkg_resources
 
-from pkg_resources import get_distribution
-
-__version__ = get_distribution("google-cloud-storage").version
+try:
+    __version__ = pkg_resources.get_distribution("google-cloud-storage").version
+except pkg_resources.DistributionNotFound:
+    __version__ = None
```

Comment:
Is there another way to set __version__?

Reply:
Yep. This is the other approach: https://github.com/googleapis/python-datastore/pull/94/files

We would have to change to using version.py instead of setup.py. 

The generator doesn't try to get the version in failure, just guards to avoid user disruption. Though maintaining version info may be useful :). Happy to go that route, but started with the 'light' touch

Reply:
This is a workaround for users who are off-roading (installing with non-pip).